app/api/events.py

```diff
@@ -27,6 +27,14 @@ class Meta:
                           schema='TicketSchema',
                           many=True,
                           type_='ticket')
+    notification = Relationship(attribute='notification',
+                          self_view='v1.event_notification',
+                          self_view_kwargs={'id': '<id>'},
+                          related_view='v1.notification_detail',
+                          related_view_kwargs={'id': '<id>'},
```

Comment:
You have to add it in __init__.py.  Check [this PR](https://github.com/fossasia/open-event-orga-server/pull/3710) for reference. Right now, the related url will be of the form /v1/notifications/<event_id> when you visit v1/events/<event_id>. Since the id is same in both places.